samples/nrf9160/modem_shell/README.rst

```diff
@@ -678,8 +726,22 @@ CONFIG_MOSH_PPP
 .. _CONFIG_MOSH_REST:
 
 CONFIG_MOSH_REST
-   Enable REST feature in modem shell
+   Enable REST client feature in modem shell
+
+.. _CONFIG_MOSH_CLOUD_REST:
 
+CONFIG_MOSH_CLOUD_REST
+   Enable nRF Cloud REST connection feature in modem shell
```

Comment:
I would drop the word "connection", because there's really no connection for REST. And "feature" could be plural, but that's not a deal breaker for me. 😄 
```suggestion
   Enable nRF Cloud REST feature in modem shell
```